digitalearthau/sync/submit_job.py

```diff
@@ -163,6 +163,8 @@ def submit(self,
             '-o', str(output_file),
             '-W', ','.join(attributes),
             '--',
+            'module use %s; ' % MODULES_PATH,
+            'module load %s; ' % MODULE_NAME,
```

Comment:
I haven't run it, but this may not work? I think qsub takes a command, not a bash string.

Reply:
From the man page: `-- executable  [arglist for executable]]`

or else specify a script path without `--`